src/h5web/visualizations/containers/RawVisContainer.tsx

```diff
@@ -9,7 +9,7 @@ function RawViscontainer(props: VisContainerProps): ReactElement {
   assertDataset(entity);
 
   const value = useDatasetValue(entity.id);
-  if (!value) {
+  if (value === undefined) {
```

Comment:
This bloody regression was back again... 😄 The scalar (and raw) vis was not display datasets with falsy values like `0` or `""`. That's why I created a mocked scalar entity with a value of `0`, and updated a feature test to look for it.